packages/ui/src/components/va-virtual-scroller/VaVirtualScroller.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div ref="wrapper" class="va-virtual-scroller">
+    <div class="va-virtual-scroller__container" :style="containerStyleComputed">
+      <div
+        ref="list"
+        role="list"
+        class="va-virtual-scroller__list"
+        :class="listClassComputed"
+        :style="listStyleComputed"
+      >
+        <template
+          v-for="(item, index) in renderBuffer"
+          :key="uniqueKey(item, index)"
+        >
+            <slot v-bind="{item, index: renderStartIndex + index}" />
+        </template>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref, computed } from 'vue'
+import pick from 'lodash/pick.js'
+import isArray from 'lodash/isArray.js'
+
+import { useEvent, useBem } from '../../composables'
+import { useVirtualScrollerSizes, useVirtualScrollerSizesProps } from './useVirtualScrollerSizes'
+
+import { warn } from '../../services/utils'
+
+export default defineComponent({
+  name: 'VaVirtualScroller',
+
+  props: {
+    ...useVirtualScrollerSizesProps,
+    items: { type: Array, required: true, validator: (v: any[]) => !!v.length },
+    bench: { type: Number, default: 10, validator: (v: number) => v >= 0 },
+    trackBy: { type: String, default: '' },
```

Comment:
Can't we reuse `trackBy` from `useSelectableList` composable?

Reply:
It's not logically suitable for this component. Also `useSelectableList` has not exactly the same what i need here.

Reply:
@aluarius hmm, maybe we can move trackBy to different composable or something?

trackBy logic fits our case very nicely, no?